scripts/actions/mobskills/batterhorn.lua

```diff
@@ -4,17 +4,19 @@
 --  Type: Physical
 --  Utsusemi/Blink absorb: 2-3 shadows
 --  Range: Melee
---  TODO: Verify damage type, number of hits requires retail verification
---  https://www.bg-wiki.com/ffxi/Locus_Wivre
 -----------------------------------
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
-    return 0
+    if target:isInfront(mob) then
+        return 0
+    end
+
+    return 1
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local numhits = 3
+    local numhits = math.random(2, 3)
```

Comment:
I don't think we should have any randoms inside numhits. I think this either means the skill has 3 hits and can miss or its an aoe that wipes shadows.

Reply:
I believe that and will happily change this but this is a thing in LOTS of mob skills

Reply:
Yeah most are just copy/paste from the wiki. I would just fix the ones you run into per family if there are any